### DISCUSSION TITLE
See changes from force push

### DISCUSSION THREAD

**Discussion by ricardoV94** [community/community#5095] [Mobile]
Doesn't seem to be possible to do this from the mobile app

**Comment by jsoref** [comment]
I was just about to file this issue. (It's still present.)

GitHub Mobile beta

1. Visit  https://github.com/cert-manager/website/pull/992
2. Scroll to a forced push entry
    ![Screenshot_20230216-075943.png](https://user-images.githubusercontent.com/2119212/219371712-c5d21c26-8ab7-4e34-b215-3cd9a419bab7.png)
3. There's no way to tap to view the changes.
4. Clicking [`force pushed`](https://github.com/cert-manager/website/compare/4c1343128520c0bce097f2e489fd0d58bcc99e45..28cea1d455dac4d676caf7ffe492a394846b2dc6) in Chrome for Android does what I expect...
